relay-server/src/actors/processor.rs

```diff
@@ -2156,6 +2174,9 @@ impl EnvelopeProcessorService {
             });
         }
 
+        // We need the event parsed in order to set the profile context on it
+        self.process_profiles(state);
```

Comment:
This will now only process profiles for events that were not dropped by dynamic sampling. Is that OK?

Reply:
Yes, we don't want to process profiles for transactions that were drop.

We still have one SDK that sends profiles by themselves (and so will set the `profile_id` in the context themselves until we can send that in the same envelope) and these profiles should still be processed since there's no dynamic sampling rules around profiles yet, right?